.circleci/config.yml

```diff
@@ -379,6 +379,12 @@ jobs:
           name: Delete cluster on GKE
           when: on_success
```

Comment:
We have this code duplicated, because is required by CircleCi flow. If some step fails they trigger on_fail path. After success they launch on_success path. These steps have special effect, that we are sure this cleaning scripts will be always executed.